vp/vp/settings.py

```diff
@@ -67,17 +67,20 @@
     },
 ]
 
+SESSION_ENGINE = 'django.contrib.sessions.backends.file'
+SESSION_FILE_PATH = os.path.join(BASE_DIR, 'session_tmp')
+
```

Comment:
Might need to add a 
```
if not os.path.exists(SESSION_FILE_PATH):
    os.makedirs(SESSION_FILE_PATH)
```
after the SESSION_FILE_PATH declaration. Looks like it barks if the directory doesn't yet exist.

Reply:
I encountered this too, but I think this setting is just for user session management, and not related to how we will persist serialized workflows. We can delete this variable entirely.

Reply:
Switched back to Django default, so removed this variable.